### PR TITLE
Remove fluid deps in fused_linear_param_grad_add_kernel.cu

### PR DESCRIPTION
**Pull request by sneaxiy**
### PR types
Others

### PR changes
OPs

### Describe
Remove fluid dependencies in `fused_linear_param_grad_add_kernel.cu`.

### CODE REVIEW

File: paddle/phi/kernels/funcs/fused_gemm_epilogue.h [hunk 1]
```diff
@@ -27,21 +27,20 @@ limitations under the License. */
...
 #include "paddle/phi/core/dense_tensor.h"
+#include "paddle/phi/core/enforce.h"
+#include "paddle/phi/core/scope_guard.h"
 #include "paddle/utils/optional.h"
 
 DECLARE_int64(cublaslt_exhaustive_search_times);
 
-namespace paddle {
-namespace operators {
+namespace phi {
```
Comment (chenwhql) [inline paddle/phi/kernels/funcs/fused_gemm_epilogue.h:43]: funcs下面的namespace需要加一层funcs，readme有写，但不太清楚，我后面再补充下示例
image

Comment (sneaxiy) [reply paddle/phi/kernels/funcs/fused_gemm_epilogue.h:43]: Done.

**Review by YuanRisheng — APPROVED** [review]

File: paddle/phi/core/scope_guard.h [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (zyfncg) [inline paddle/phi/core/scope_guard.h:1]: 2021 -> 2023

**Review by zyfncg — APPROVED** [review]

File: paddle/phi/core/scope_guard.h (hunk 2 above)
Comment (sneaxiy) [reply paddle/phi/core/scope_guard.h:1]: Thanks. Would fix in the next PR.

**Review by luotao1 — APPROVED** [review]
LGTM for paddle_enforce

**Pull request merged**